NuGet.Config

```diff
@@ -7,11 +7,20 @@
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
-    <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption%40Local/nuget/v3/index.json" />
+    <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption/nuget/v3/index.json" />
```

Comment:
I think you may have unintentionally removed the `%40Local` view from the package source. It looks like the [restore operation failed in CI](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9876591&view=logs&j=d80b2849-30d4-52dd-74cd-d6536ba98fe0&t=34253729-59a1-51a0-5e34-6fa41cb200bd&l=21) for this branch because the consumption feed challenged the client with a 401 error, and we don't run the restore in interactive mode there. My suggestion is to add local view back to the feed. Run the restore operation locally in interactive mode to ingest packages from upstream sources to the public local view.

Reply:
It's possible I lost this during a merge too, reverted it.